mmdet/core/bbox/match_costs/match_cost.py

```diff
@@ -265,7 +271,12 @@ def binary_mask_dice_loss(self, mask_preds, gt_masks):
         mask_preds = mask_preds.flatten(1)
         gt_masks = gt_masks.flatten(1).float()
         numerator = 2 * torch.einsum('nc,mc->nm', mask_preds, gt_masks)
-        denominator = mask_preds.sum(-1)[:, None] + gt_masks.sum(-1)[None, :]
+        if self.naive_dice:
+            denominator = mask_preds.sum(-1)[:, None] + \
+                gt_masks.sum(-1)[None, :]
+        else:
+            denominator = torch.sum(mask_preds * mask_preds, 1)[:, None] + \
```

Comment:
torch.sum(mask_preds * mask_preds, 1) -> mask_preds.pow(2).sum(1), also update that for gt_masks